src/store/features/board/thunks.ts

```diff
@@ -296,7 +296,7 @@ export const deleteBoard = createAsyncThunk<
 >("board/deleteBoard", async (_payload, {dispatch, getState}) => {
   const {id} = getState().board.data!;
   retryable(() => API.deleteBoard(id), dispatch, deleteBoard, "deleteBoard").then(() => {
-    document.location.pathname = "/";
+    redirectToBoardDeletedPage();
   });
```

Comment:
if the owner also receives the `BOARD_DELETED` event above, this line could even be omitted, couldn't it?